alert/log/store.go

```diff
@@ -320,7 +321,12 @@ func (db *DB) logAny(ctx context.Context, tx *sql.Tx, insertStmt *sql.Stmt, id i
 			}
 			if _type == TypeNoNotificationSent {
 				// no CMID for no notification sent
-				r.subject.classifier = "no immediate rule"
+				cfg := config.FromContext(ctx)
+				if !cfg.Twilio.Enable {
+					r.subject.classifier = "Twilio Disabled"
+				} else {
+					r.subject.classifier = "no immediate rule"
+				}
 				break
```

Comment:
We don't need this anymore do we?

Reply:
Since `TypeNoNotificationSent` is no longer used for the Twilio disabled code, we can revert this to the previous code